buildscripts/builddrivers.py

```diff
@@ -0,0 +1,291 @@
+                    os.startfile(os.path.join(root_dir, 'php-sdk', logfile))
+                os.chdir(work_dir)    
+                break
+
+            # Only ask when building locally
+            if self.local:               
+                choice = input("Rebuild the same configuration(yes) or quit (no) [yes/no]: ")
+
+                if choice.lower() == 'yes' or choice.lower() == 'y' or choice.lower() == '':
+                    print('Rebuilding drivers...')
+                    self.make_clean = False
+                    self.rebuild = True
+                    self.util.remove_prev_build(root_dir)
+                else:
+                    quit = True
+            else:
+                quit = True
+            
+            os.chdir(work_dir)    
+
+def validate_input(question, values):
+    """Return the user selected value, and it must be valid based on *values*."""
+    while True:
+        options = values.split('/') 
+        prompt = '[' + values + ']'
+        value = input(question + prompt + ': ')
+        value = value.lower()
+        if not value in options:
+            print("An invalid choice is entered. Choose from", prompt)
+        else:
+            break
+    return value
+
+################################### Main Function ###################################
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v', '--PHPVER', help="PHP version, e.g. 7.1.*, 7.2.* etc.")
+    parser.add_argument('-a', '--ARCH', choices=['x64', 'x86'])
+    parser.add_argument('-t', '--THREAD', choices=['nts', 'ts'])
+    parser.add_argument('-d', '--DRIVER', choices=['all', 'sqlsrv', 'pdo_sqlsrv'])
+    parser.add_argument('-m', '--DEBUG', default='no', choices=['yes', 'no'], help="enable debug mode")
+    parser.add_argument('-r', '--REPO', default='Microsoft', help="GitHub repository")
+    parser.add_argument('-b', '--BRANCH', default='dev', help="GitHub repository branch")
+    parser.add_argument('-g', '--GITHUB', default='yes', help="get source from GitHub or not")
+    parser.add_argument('-k', '--PACKAGE', default='Latest', help="the package name for the drivers")
+    parser.add_argument('-p', '--PATH', default=None, help="the remote destination for the drivers (do not use this when building locally)")
+
+    args = parser.parse_args()
+
+    phpver = args.PHPVER
+    arch = args.ARCH
+    thread = args.THREAD
+    driver = args.DRIVER
+    debug = args.DEBUG == 'yes'
+    repo = args.REPO
+    branch = args.BRANCH
+    download = args.GITHUB.lower() == 'yes'
+    path = args.PATH
+    package = args.PACKAGE
+
+    if phpver is None:
+        # assuming it is building drivers locally when required to prompt
+        # thus will not prompt for drivers' destination path, which is None by default
+        phpver = input("PHP Version (e.g. 7.1.* or 7.2.*): ")
+        arch_version = input("Want to build 64-bit [y/n]: ")
```

Comment:
the words "Want to build" is unnecessary" and is inconsistent with the rest of the UI